pandas_todo.md

```diff
@@ -69,14 +69,6 @@ Repro:
         return GraphTraversal(self.vertices, self.edges, new_paths,
                               new_path_col_types, self._aliases)
 ```
-2. Attempt to call the `in_()` method:
```

Comment:
Can you move this entire subsection (starting at the subhead `## Bug in merge` on line 42) to a file `pandas_todo.md` in the new Gremlin project? We might need to come back and fix this bug later as we move forward on the Gremlin work.

Reply:
done